app/helpers/translations_helper.rb

```diff
@@ -27,3 +27,34 @@ def translate_boolean(bool)
     t(bool ? "reply_yes" : "reply_no")
   end
 end
+
+module SimpleForm
+  class FormBuilder
+    include TranslationsHelper
+
+    def translatable_inputs(&block)
+      out = %Q{
+        <div class="languages">
+          <input class="deleted-locales" name="#{object.model_name.singular}[deleted_locales][]" type="hidden" />
+      }.html_safe
+      object.used_locales_or_current_locale.each do |l|
+        out += %Q{
+          <div class="language-block" data-locale="#{l}">
+            <a class="remove-language" href="#">#{I18n.t('common.remove')}</a>
+        }.html_safe
+        out += input :"locale_#{l}", label: false,
+          collection: locale_options,
+          input_html: {class: 'locale'}, include_blank: false
+
+        out += template.capture { yield l }
+
+        out += "</div>".html_safe
```

Comment:
This is a big no-no I think because there can be arbitrary content inside the input tag, so you shouldn't be using html_safe. You shouldn't need it if you use `content_tag` and such. Let's discuss.